packages/frontend/engine/src/__tests__/frame_pacing.test.ts

```
@@ -0,0 +1,261 @@
+// - AC-3: Buffer safety (copy before recycle)
+// - AC-4: Unprojection round-trip identity
+
+import { describe, expect, it } from 'bun:test';
+
+// ---------------------------------------------------------------------------
+// AC-1: Fixed-timestep accumulator
+// ---------------------------------------------------------------------------
+
+describe('C-380 AC-1: Fixed-timestep accumulator', () => {
+  it('consumes exactly one step when elapsed equals step size', () => {
+    const FIXED_STEP_MS = 1000 / 60; // ~16.667
```

Biome (2.5.7)

[error] This const name should be in camelCase or PascalCase.

(lint/style/useNamingConvention)